content/html/concepts/attributes/attributes.md

```diff
@@ -30,29 +30,27 @@ So for example, the `<img>` image element has a mandatory `src` attribute that c
 - `src` is the attribute name.
 - `codecademy_logo.png` is the attribute value.
 
-All HTML elements can have attributes. Multiple of them, too!
+All HTML elements can have attributes; many have more than one.
 
 ```html
 <img src="nyan_cat.gif" width="500" height="600" />
 ```
 
-**Note:** Attribute names are case insensitive, but it is a good practice to keep HTML markup lowercase.
+**Note:** Attribute names are not case-sensitive, but it is good practise to write HTML markup in lowercase.
 
 ## Unique ID Attributes
 
-In HTML, specific and unique `id` attributes can be assigned to different elements in order to differentiate between them.
+Specific and unique `id` attributes can be assigned to elements to differentiate them, and `id` values can be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element alone, without affecting others of the same type.
```

Comment:
```suggestion
`id` attributes are specific and unique. They can be assigned to differentiate elements. They can also be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element without affecting similar tags.
```

Reply:
Maybe we could be even clearer/more succinct here?

> The `id` attribute is generally used to differentiate elements. It is commonly referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element without affecting other elements of the same tag. Its value should be unique.

Reply:
I think it's more so that the `id` attribute is used to identify elements. Other than that, loving that explanation. Let's change it to what you wrote here.

Reply:
Awesome, thanks!